### PR TITLE
[WIP] IZE-673 allow empty maps in viper

### PR DESCRIPTION
**Pull request by psihachina**
## Change:
allow empty maps in viper

## Test:
[asciicast](https://asciinema.org/a/PCCGs5YIYNmYzdFkpCLoCqzrc)

### CODE REVIEW

**Review by igorkotof — APPROVED** [review]
LGMT

**Review by AutomationD — CHANGES_REQUESTED** [review]

**Comment by AutomationD** [comment]
> LGMT

TMLG

File: go.mod [hunk 1]
```diff
@@ -94,33 +97,35 @@ require (
...
+	golang.org/x/sys v0.0.0-20220728004956-3c1f35247d10 // indirect
 	golang.org/x/time v0.0.0-20220224211638-0e9765cccd65 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
 replace github.com/spf13/pflag => github.com/cornfeedhobo/pflag v1.1.0
+
+// spf13/viper ignore empty map
```
Comment (AutomationD) [inline go.mod:128]: Pls fix grammar

**Review by AutomationD — COMMENTED** [review]
Are there any risks with this PR and our fork?

**Pull request closed without merging**